lib/components/buttons/inverted_button.js

```diff
@@ -79,8 +77,6 @@ const styles = StyleSheet.create({
     flex: 1,
     justifyContent: 'center',
     alignItems: 'center',
-    marginLeft: sideMargin,
```

Comment:
This makes it such that the button styles its own contents but not its overall width, height, or margins.